.template-lintrc.js

```diff
@@ -1,54 +1,5 @@
-'use strict'
+'use strict';
```

Comment:
The overwritten configuration was the common one across all our add-ons, we should keep it.

Reply:
:+1: I revert it